src/app/page-content/books/book/book.component.ts

```diff
@@ -13,20 +14,21 @@ export class BookComponent {
   @Input()
   book: BookDto = {} as BookDto;
 
-  isEditMode: boolean = false;
-  bookDetailsForm: FormGroup = new FormGroup({})
+  @Output()
+  isEditMode$: Subject<boolean> = new Subject<boolean>();
+  bookDetailsForm: FormGroup = new FormGroup({});
 
   constructor(private pageContentBaseService: PageContentBaseService) { }
 
   onEdit(title: string){
-    this.isEditMode = true;
+    this.isEditMode$.next(true);
     this.bookDetailsForm = new FormGroup({
       'title': new FormControl(title),
     });
   }
 
   onSave(id: string){
     this.pageContentBaseService.renameBook(id, this.bookDetailsForm.controls['title'].value);
-    this.isEditMode = false;
+    this.isEditMode$.next(false);
```

Comment:
to jeżeli jest wewnątrz komponenty mogło by być normalną zmienną boolean, Subject jest chyba niepotrzebny